setup.py

```diff
@@ -47,13 +47,15 @@ def build_extension(self, ext):
 
 setup(
     name="crypto_cpp_py",
-    version="1.0.1",
+    version="1.0.2",
     description="This is a packaged crypto-cpp program",
     author="Wojciech Szymczyk",
     author_email="wojciech.szymczyk@swmansion.com",
     url="https://github.com/software-mansion-labs/crypto-cpp-py.git",
     long_description=long_description,
     long_description_content_type="text/x-rst",
+    install_requires=["cairo_lang==0.8.2.1"],
```

Comment:
Shouldn't we allow for any cairo-lang > 0.8.2.1 instead? Or is there anything in this package that would possibly break with cairo-lang update?

Reply:
We could, but I'd like to keep it in sync with the version pinned by starknet.py; I just didn't want to cause any possible conflicts. I am open to discuss however.

Reply:
Alright, we can keep it in sync manually. Still perhaps it would be good to have some sort of check for new cairo-lang version (like dependabot or something like that?).

Reply:
Hmm, now that I think about it - maybe it makes sense to just not pin the version at all? The module only makes use of signatures from cairo package, that should not change generally (I think), so the module could in fact just take any latest version (or the one pinned by, for example, starknet.py), and that would save us some future work (or rather time to rebuild). What do you think?

Reply:
I actually don't know now: cairo-lang is still in development and likes to do breaking changes between releases (though it's mostly apis). That could cause some unexpected problems down the road. 

Am I correctly understanding that if we pinned the version to the exact version of cairo-lang (eg `"cairo_lang==0.8.2.1"`), we would have to rebuild this whole package every time cairo lang is bumped? If that's correct, it would mostly negate benefits of separating crypto-cpp from starknet.py, since there would be no time saved.

Maybe we could just implement some workflow for verifying if this package still works when new version of cairo-lang is released and then manually modify `install_requires`?